Task2/js/app.js

```diff
@@ -0,0 +1,24 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const container = document.querySelector('.zombies');
+
+    for (let i = 0; i < zombies.length; i++) {
+        const zombie = document.createElement('div');
+
+        zombie.classList.add('zombie');
+
+        zombie.addEventListener('click', () => {
+            zombie.classList.remove(zombie.classList[zombie.classList.length - 1]);
```

Comment:
wtf?
you should add "zombie" class by default on zombieElement
then you should add "dead" class on click